src/update.rs

```diff
@@ -43,27 +44,50 @@ pub struct RevealRevocationSecretKey {
 pub struct Party0 {
     x_self: OwnershipKeyPair,
     X_other: OwnershipPublicKey,
-    TX_f: FundingTransaction,
+    TX_f_body: FundingTransaction,
     current_state: ChannelState,
     revoked_states: Vec<RevokedState>,
 }
 
 impl Party0 {
+    pub fn new(party: create::Party6) -> Self {
```

Comment:
Some doc to explain PartyN would be good. Is that coming from the paper?

Reply:
I wasn't sure what to say about this first state because it's such a generic one. Maybe I'll add that it represents any state for which the channel is still open and an arbitrary number of updates have taken place.

> Is that coming from the paper?

Nope.

Reply:
Why do you call it `Party` if it represents a state?

Reply:
Well, we have `Alice` and `Bob` which are "parties", so I thought the general one could be `Party`.

We can rename all this though. I'm definitely not convinced about `Party`.

Reply:
So `Party6` means "State number 6 (counting from zero) of a given party"?

Just call it `State 6` instead, no?

Reply:
Ok, this is roted :)